composer.json

```
@@ -16,10 +16,10 @@
         }
     },
     "require": {
-        "illuminate/translation": "^8.0||^6.0"
+        "illuminate/translation": "^9.0||^8.0||^6.0"
```

We'll also reorder them:
```suggestion
        "illuminate/translation": "^6.0||^8.0||^9.0"
```

---

Noted.